_includes/components/slider.html

```diff
@@ -1,27 +1,30 @@
-{% assign event = include.event %}
-{% assign impressions = nil | arrayify %}
-{% for file in site.static_files %}
-  {% assign isImpression = file.path | startswith: "/resources/images/impressions/" %}
-  {% if isImpression %}
-    {% assign fileArray = file | arrayify %}
-    {% assign impressions = impressions | concat: fileArray %}
-  {% endif %}
-{% endfor %}
-{% if impressions.first %}
+{% capture imageQuery %}
+  SELECT ?image ?description ?link
+  WHERE {
+    ?image rdf:type foaf:Image ;
```

Comment:
This is a very general query that would potentially return all images in the graph. Could you please attach it to the `<https://2022.dataweek.org/>` resource with some meaningful property?

For the IRIs you have used the github namespace. Please use the dataweek namespace (https://github.com/AKSW/leipzig.dataweek.de-model/commit/a08d80debf85582864f82c2e1288dc0ad091a586).